saleor/core/utils/random_data.py

```diff
@@ -456,9 +457,10 @@ def create_fake_order():
 
     payment = create_payment(delivery_group)
     if payment.status == PaymentStatus.CONFIRMED:
-        order.change_status(OrderStatus.FULLY_PAID)
+        order.status = OrderStatus.FULLY_PAID
         if random.choice([True, False]):
-            order.change_status(OrderStatus.SHIPPED)
+            order.status = OrderStatus.SHIPPED
+        order.save()
```

Comment:
You are already saving order on line 449, is it necessary for creating delivery group?

Reply:
It is, because order must be saved before payment or delivery group is created.